types/office-js-preview/index.d.ts

```diff
@@ -9573,6 +9573,124 @@ declare namespace Office {
              */
             Removed = "removed"
         }
+        /**
+         * Specifies the category color.
+         * 
+         * [Api set: Mailbox Preview]
+         * 
+         * @remarks
+         * <table>
+         *   <tr><td>{@link https://docs.microsoft.com/outlook/add-ins/#extension-points | Applicable Outlook mode}</td><td>Compose or Read</td></tr>
+         * </table>
+         * 
+         * @beta
+         */
+        enum CategoryColor {
+            /**
+             * Default color
+             */
+            None,
+            /**
+             * Red
```

Comment:
Would it be possible to add the hex value to these colors? Right now, the comments add little value.

Reply:
I don't think so. If I understand it correctly, the actual color displayed may vary by platform. @exextoc: Ankush, can you confirm this? Thanks.

Reply:
Confirmed with Ankush.